templates/workspace_services/azureml/porter.yaml

```diff
@@ -137,6 +143,62 @@ install:
         auth_client_id: ${ bundle.credentials.auth_client_id }
         auth_client_secret: ${ bundle.credentials.auth_client_secret }
         auth_tenant_id: ${ bundle.credentials.auth_tenant_id }
+        public_access_enabled: true
+        arm_environment: ${ bundle.parameters.arm_environment }
+        azure_environment: ${ bundle.parameters.azure_environment }
```

Comment:
Apologies if this isn't strictly related to your PR, just wondering what the difference between arm_environment and azure_environment is? Its a little confusing

Reply:
yes - not sure myself, was like that when i get ere :)